From 0481454a4d9e05e97b3ff4591d5cca70c1a43544 Mon Sep 17 00:00:00 2001
From: Jeff Lance <jlance1@ti.com>
Date: Tue, 29 Nov 2011 10:30:53 -0600
Subject: [PATCH 0645/1092] arm:omap:am335x: Enable Oprofile support on AM335x

This enables support for oProfile on AM335x.

Enabling Oprofile does not impact kernel performance since
Oprofile is dormant unless a profiling session is initiated.

Signed-off-by: Jeff Lance <j-lance1@ti.com>
Signed-off-by: Hebbar, Gururaja <gururaja.hebbar@ti.com>
Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 arch/arm/configs/am335x_evm_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/am335x_evm_defconfig b/arch/arm/configs/am335x_evm_defconfig
index 06c060e..88804ca 100644
--- a/arch/arm/configs/am335x_evm_defconfig
+++ b/arch/arm/configs/am335x_evm_defconfig
@@ -123,7 +123,8 @@ CONFIG_VM_EVENT_COUNTERS=y
 CONFIG_COMPAT_BRK=y
 CONFIG_SLAB=y
 # CONFIG_SLUB is not set
-# CONFIG_PROFILING is not set
+CONFIG_PROFILING=y
+CONFIG_OPROFILE=y
 CONFIG_HAVE_OPROFILE=y
 # CONFIG_KPROBES is not set
 CONFIG_HAVE_KPROBES=y
-- 
1.7.11.2

